src/library/F3DOptions.cxx

```diff
@@ -730,6 +730,8 @@ F3DOptions F3DOptionsParser::GetOptionsFromCommandLine(std::vector<std::string>&
 //----------------------------------------------------------------------------
 void F3DOptionsParser::ConvertToNewAPI(const F3DOptions& oldOptions, f3d::options* newOptions)
 {
+  f3d::options::initialize(*newOptions);
```

Comment:
It should be in `options::F3DInternals` constructor I think. Any reason to call it explicitely?

Reply:
Maybe someone do not want to initialize ? currently F3D do not require using it.

Reply:
Then how to determine if a user specifies an option but it's not valid? The map should be initialized in the constructor imo, then any call to `set` should verify the key is already there.

Reply:
I think you proposed behavior is indeed what we want. If we need more flexibility we can improve that in the future